LearningMissionSimulation/PlaygroundGarush.cs

```diff
@@ -16,10 +17,18 @@ public class PlaygroundGarush : ISimulation
         Dictionary<Guid, Module> moduleDictionary = new Dictionary<Guid, Module>();
         List<Guid> moduleIdList = new List<Guid>();
 
+        public PlaygroundGarush(ReportType reportType)
+        {
+            this.ReportType = reportType;
+        }
+        public ReportType ReportType { get => _reportType; set => _reportType = value; }
+
         public void CreateAccounts(int accountCount)
         {
+            string action = " create account ";
```

Comment:
Remove trailing spaces in "create account". Also you have to decide you use plural or singular nouns: "create acount" vs "create accounts". Capital vs lowercase: "Create account" or "create account". And you have to follow the same rule everywhere. Otherwise it does not look consistent.